app/views/forgot_passwords/new.html.haml

```diff
@@ -0,0 +1,13 @@
+%section.forgot_password.container
+  .row
+    .col-sm-10.col-bg-offset-1
+      = form_tag forgot_passwords_path, class: "sign_in" do 
+        %header
+          %h1 Forgot Password?
+          %p We will send you an email with a link that you can use to reset your password.
+        .form-group
+          = label_tag :email, "Email Address"
+          .row.col-sm-4
+            = email_field_tag :email
```

Comment:
`= email_field_tag :email, nil, class: "form-control"` this is how it looks in my project. I think you have to add `class: "form-control"`